app/controllers/miq_ae_class_controller.rb

```diff
@@ -258,10 +258,6 @@ def find_existing_node(parents)
     existing_node
   end
 
-  def build_ae_tree
```

Comment:
are all of these deletes because the parent method call is the same?
Or because they are no longer referenced?

(via/ "delete unnecessary build*tree methods")

Reply:
Yup, no longer referenced via the `method(sting).call`, except reports